### PR TITLE
Pull Request Tips

### PR DESCRIPTION
**Pull request by WillAyd**
I think these issues pop up on a good deal of PRs but aren't explicitly stated anywhere, so adding a few bullet points to the contributing guide might help

### CODE REVIEW

File: doc/source/development/contributing.rst [hunk 1]
```diff
@@ -1525,3 +1525,18 @@ The branch will still exist on GitHub, so to delete it there do::
...
+Tips for a successful Pull Request
+==================================
+
+If you have made it to the `Review your code`_ phase one of the core contributors may
+take a look. Please note however that a handful of people are responsible for reviewing
+all of the contributions, which can often lead to bottlenecks.
+
+To improve the chances of your pull request being reviewed, you should:
+
+  - **Ensure you have appropriate tests**. These should be the first part of any PR
```
Comment (jorisvandenbossche) [inline doc/source/development/contributing.rst]: RST gotcha: bullet points require no indentation

File: doc/source/development/contributing.rst [hunk 2]
```diff
@@ -1525,3 +1525,18 @@ The branch will still exist on GitHub, so to delete it there do::
     git push origin --delete shiny-new-feature
 
 .. _Gitter: https://gitter.im/pydata/pandas
+
+
+Tips for a successful Pull Request
+==================================
+
+If you have made it to the `Review your code`_ phase one of the core contributors may
```
Comment (jorisvandenbossche) [inline doc/source/development/contributing.rst]:
```suggestion
If you have made it to the `Review your code`_ phase, one of the core contributors may
```

**Review by jorisvandenbossche — COMMENTED** [review]
Good idea!

**Comment by simonjayhawkins** [comment]
yep good idea. maybe also something about raising an issue for discussion before submitting a pull request if there is not already an issue on the subject.

**Review by gfyoung — APPROVED** [review]

**Comment by gfyoung** [comment]
Follow-up: I would link to this document in the PR template.

**Review by simonjayhawkins — APPROVED** [review]

**Comment by jreback** [comment]
thanks @WillAyd 

if you can followup with @gfyoung suggestion would be great.

**Pull request merged**